export const throwError = (e) => {
  throw e;
};
